mne/beamformer/_compute_beamformer.py

```diff
@@ -427,7 +427,7 @@ def _compute_beamformer(G, Cm, reg, n_orient, weight_norm, pick_ori,
             # Estimate noise level based on covariance matrix, taking the
             # first eigenvalue that falls outside the signal subspace or the
             # loading factor used during regularization, whichever is largest.
-            if rank >= len(Cm):
+            if rank > len(Cm):
```

Comment:
This would now only come in play if the user manually specified a rank higher than the cov matrix dimensions. Can we see such a thing happening / being useful?

Reply:
thoughts, @sarangnemo?

Reply:
indeed, I'm confused. What does a rank higher than the data dimensionality mean